dashboard/src/pages/ingest/sips/index.vue

```diff
@@ -33,6 +35,14 @@ import IconSIPs from "~icons/octicon/package-dependencies-24";
 const authStore = useAuthStore();
 const layoutStore = useLayoutStore();
 const sipStore = useSipStore();
+const userStore = useUserStore();
+
+const uploaderEl = ref<HTMLElement | null>(null);
+const uploaderDD = ref<Dropdown | null>(null);
+const uploaderDDLabel = ref("Uploaded by");
+
+// Increase the default limit for the uploader dropdown to 100 users.
+userStore.page.limit = 100;
```

Comment:
What's the point of doing it from here? This will change it permanently after visiting this page.

Reply:
I don't think it actually matters at this point because this is the only page using the userStore, and there is no way to adjust the limit from the UI at this point.   In any case, I've changed the default limit in the user store instead.